### PR TITLE
add siamrpn and siammask code

### PR DESCRIPTION
**Pull request by xbsu**
update VOT models with SiamRPN and SiamMask

### CODE REVIEW

**Review by bilylee — COMMENTED** [review]
PaddleCV/tracking/ltr/data/anchor.py 文件是否需要加 pysot 的 license？

**Review by bilylee — COMMENTED** [review]
是否还缺少跟踪测试时使用的 tracker 和 parameters ？

File: PaddleCV/tracking/ltr/models/siam/neck.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import paddle.fluid as fluid
```
Comment (bilylee) [inline PaddleCV/tracking/ltr/models/siam/neck.py:1]: 是否需要加 pysot 的 license？

File: PaddleCV/tracking/ltr/models/siam/xcorr.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import paddle.fluid as fluid
```
Comment (bilylee) [inline PaddleCV/tracking/ltr/models/siam/xcorr.py:1]: 是否需要加 pysot license？

File: PaddleCV/tracking/ltr/actors/siam.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
+from . import BaseActor
```
Comment (yuanyc06) [inline PaddleCV/tracking/ltr/actors/siam.py:1]: 每个文件开头统一加上一段固定的copyright吧，类似这种：
https://github.com/PaddlePaddle/models/blob/develop/PaddleCV/PaddleDetection/ppdet/modeling/model_input.py

Comment (qingqing01) [reply PaddleCV/tracking/ltr/actors/siam.py:1]: @yuanyc06 tracking代码里整体都没加，可以不加了

File: PaddleCV/tracking/ltr/models/loss.py [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+    pred = fluid.layers.reshape(pred, [b, 2, -1, h, w])
+    pred = fluid.layers.transpose(pred, [0, 2, 3, 4, 1])
+    pred = fluid.layers.reshape(pred, [-1, 2])
+    label = fluid.layers.reshape(label, [-1, 1])
+    loss = fluid.layers.softmax_with_cross_entropy(
+        logits = pred, 
+        label = label,
+        ignore_index = -1)
+    return fluid.layers.mean(loss)
+'''
```
Comment (qingqing01) [inline PaddleCV/tracking/ltr/models/loss.py:17]: remove un-used code

File: PaddleCV/tracking/ltr/models/siam/neck.py (hunk 1 above)
Comment (qingqing01) [reply PaddleCV/tracking/ltr/models/siam/neck.py:1]: 这个文件是paddle相关模型，看着没引用pysort吧

**Review by qingqing01 — COMMENTED** [review]
Also need to update README.md

File: PaddleCV/tracking/ltr/models/siam/neck.py (hunk 1 above)
Comment (bilylee) [reply PaddleCV/tracking/ltr/models/siam/neck.py:1]: https://github.com/STVIR/pysot/blob/master/pysot/models/neck/neck.py
与这里的代码结构是类似的
